.vuepress/docs-sidebar.js

```diff
@@ -142,7 +142,7 @@ module.exports = [
     children: [
       ['developer/', 'Overview & Introduction'],
       'developer/guidelines',
-      'developer/bindings/',
```

Comment:
I would definitely suggest to keep the "bindings" menu in place, just as we have dedicated entries for other add-on types like  automation addons, transformation services, voice add-ons etc.
What would imho make sense here is to add a new menu entry for everything that generally applies to all add-ons and this new page could then link to the different specific add-on pages as well.

Reply:
The problem is that we don't have content for that page. Duplicating what is written on the general page makes no sense.

Reply:
Well, a few parts could be moved from the binding page to the add-on page, like the description about the addon.xml, the info on how to add a logo for the add-on and similar.

Reply:
After I did that, there was no content left :-)

Reply:
Hm, I see your point, but I am a bit puzzled, because on https://www.openhab.org/docs/developer/bindings/ I see a lot more content about how to develop a binding. You seem to have moved all the files that contribute to this page to `/developer/addon`.

Reply:
Sometimes I'm a bit slow... Is this version better?

Reply:
Yes, thanks - that look better!